https://bugs.gentoo.org/938967
https://github.com/mquinson/po4a/commit/28fe52651eb8096d97d6bd3a97b3168522ba5306 (backport)

From 45c835a04a388940c627f0c6b8c26554c0ee2b14 Mon Sep 17 00:00:00 2001
From: Martin Quinson <martin.quinson@ens-rennes.fr>
Date: Fri, 12 Jul 2024 10:21:34 +0200
Subject: [PATCH] Fix failures with DynaLoader on Perl 5.40

Thanks to dstoecker for the patch provided in the OpenSuse package,
and to ana for pointing me to it.

Fixes https://github.com/mquinson/po4a/issues/508

(cherry picked from commit 28fe52651eb8096d97d6bd3a97b3168522ba5306)
---
 lib/Locale/Po4a/TransTractor.pm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/Locale/Po4a/TransTractor.pm b/lib/Locale/Po4a/TransTractor.pm
index b83dd25b..68296cb1 100644
--- a/lib/Locale/Po4a/TransTractor.pm
+++ b/lib/Locale/Po4a/TransTractor.pm
@@ -5,6 +5,8 @@ require Exporter;
 package Locale::Po4a::TransTractor;
 use DynaLoader;
 
+sub import { }
+
 use 5.006;
 use strict;
 use warnings;
